include/fast_float/ascii_number.h

```diff
@@ -232,6 +232,39 @@ loop_parse_if_eight_digits(char const *&p, char const *const pend,
   }
 }
 
+// credit @realtimechris
+fastfloat_really_inline constexpr bool
+parse_if_eight_digits_unrolled(const char *&string, size_t &value) {
+  constexpr size_t byte_mask = ~size_t(0) / 255ull;
+  constexpr size_t msb_mask = byte_mask * 128ull;
+  constexpr size_t threshold_byte_mask = byte_mask * (127ull - 9ull);
+  constexpr size_t mask = 0x000000FF000000FFull;
+  constexpr size_t mul1 = 0x000F424000000064ull;
+  constexpr size_t mul2 = 0x0000271000000001ull;
+  size_t value_new = read8_to_u64(string) - 0x3030303030303030;
+  if (!(((value_new + threshold_byte_mask) | value_new) & msb_mask)) {
```

Comment:
`value_new + threshold_byte_mask` leads me to think `threshold_byte_mask` isn't a byte mask at all